lldb/test/API/commands/expression/import-std-module/deque-dbg-info-content/TestDbgInfoContentDequeFromStdModule.py

```diff
@@ -10,7 +10,7 @@
 class TestDbgInfoContentDeque(TestBase):
     @add_test_categories(["libc++"])
     @skipIf(compiler=no_match("clang"))
-    @skipIf(compiler="clang", compiler_version=["<", "12.0"])
+    @skipIf(compiler="clang", compiler_version=["<", "12.0"], oslist=["linux"])
```

Comment:
Are these "and" or "or"? You may need to do this instead:

```suggestion
    @skipIf(compiler="clang", compiler_version=["<", "12.0"])
    @skipIf(oslist=["linux"])
```

Reply:
I was hoping for "and" but this is better solution, I'll change the other tests to match.